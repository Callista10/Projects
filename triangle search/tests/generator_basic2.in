-N=3 -mindist=3 -rseed=1