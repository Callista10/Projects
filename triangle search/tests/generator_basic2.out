-36.56, 34.74
26.38, -24.49
-0.46, -5.05
